### PR TITLE
Add optional source parameters to bypass action selection and always jump straight to installation/info…

### PR DESCRIPTION
**Pull request by Thomaash**

### CODE REVIEW

**Comment by andreyorst** [comment]
I think we can merge this. @agur4ik your opinion?

**Comment by flameshikari** [comment]
@agur4ik, just do it!

**Comment by agura-lex** [comment]
Thank you for your pull request. I'll look into this more closely on the next week or so as now I'm busy. I wanted to fix some things but maybe I'll do it after merging this.

File: usr/share/doc/find-the-command/ftc.zsh [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+_cnf_actions=('install' 'info' 'list files' 'list files (paged)')
+
 # Parse options
 for opt in $*
 do
     case $opt in
         noprompt) cnf_noprompt=1 ;;
         su) cnf_force_su=1 ;;
         quiet) cnf_verbose=0 ;;
+        install) cnf_action=$_cnf_actions[1] ;;
```
Comment (agura-lex) [inline usr/share/doc/find-the-command/ftc.zsh:13]: I don't really understand the point of these

This case switch is for some basic configuration via options appended to the `source *.zsh` command

Comment (Thomaash) [reply usr/share/doc/find-the-command/ftc.zsh:13]: This adds the option `install` which skips [the action selection](https://github.com/agur4ik/find-the-command/pull/3/files#diff-8b576982a090b417663c3a0676b58f2fR82) and jumps straight to package installation just like if the user selected install.

Comment (agura-lex) [reply usr/share/doc/find-the-command/ftc.zsh:13]: Okay, now I get it.

**Pull request merged**